cmd/typed-errors.go

```diff
@@ -90,5 +90,8 @@ var errGroupNotEmpty = errors.New("Specified group is not empty - cannot remove
 // error returned in IAM subsystem when policy doesn't exist.
 var errNoSuchPolicy = errors.New("Specified canned policy does not exist")
 
+// error returned in IAM subsystem when an external users systems is configured.
+var errIAMActionNotPossible = errors.New("Specified IAM action is not possible under the current configuration")
```

Comment:
Can we keep it `errIAMActionNotAllowed` ?  like ErrMethodNotAllowed?

Reply:
:+1: